dist/htmlminifier.js

```diff
@@ -285,8 +237,30 @@
       parseEndTag();
     }
 
-    function parseStartTag( tag, tagName, rest, unary ) {
-      var unarySlash = false;
+    function parseStartTag(input) {
+      var start = input.match(startTagOpen);
+      if (start) {
+        var match = {
```

Comment:
Can't we just transpile the browser version? I have no experience with ES6, but it should be doable judging by other projects.